### ISSUE TITLE
Access the CFG

### ISSUE THREAD

**Issue by golsch** [WebAssembly/binaryen#2896]
I am interested in CFG, is there a way to get or print a CGF for a module?

**Comment by aheejin** [comment]
I don't think we have one, but we can create CFG data structure with [this class](https://github.com/WebAssembly/binaryen/blob/6ec03c5c0c664660a88fdb61ac1d3bd1f8b14771/src/cfg/cfg-traversal.h#L39). I think you can write a code to write a graph using this data structure.
